### PR TITLE
[Testing] Enabling ported UITests from Xamarin.UITests to Appium - 19

### PR DESCRIPTION
**Pull request by ghost**
### Description of Change

This PR focuses on re-enabling and updating following tests from the Xamarin.UITest framework to be compatible with Appium. The tests, previously commented out, are reviewed, and modified to ensure they are functional with the Appium framework. We are going to enable tests in blocks in different PRs. This is the 19th group of tests enabled.

Note: All the test cases are ensuring the color changes, so replaced the App.Screenshot() into VerifyScreenshot().

### TestCases

- **Issue3798**
- **Bugzilla30166**
- **Issue1323**
- **Issue3342**
- **Issue7396**

### CODE REVIEW

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue7396.cs [hunk 1]
```diff
@@ -10,19 +10,21 @@ public Issue7396(TestDevice testDevice) : base(testDevice)
...
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void BottomTabColorTest()
+	{
+		//7396 Issue | Shell: Setting Shell.BackgroundColor overrides all colors of TabBar
+		App.WaitForElement(CreateBottomTabButton);
+		App.Tap(CreateBottomTabButton);
+		App.Tap(CreateBottomTabButton);
+		App.Tap(ChangeShellColorButton);
+		VerifyScreenshot();
```
Comment (jsuarezruiz) [inline src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue7396.cs:28]: The Windows snapshots are pending and available in the latest build.
image

Could you commit the images?

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue7396.cs (hunk 1 above)
Comment (jsuarezruiz) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue7396.cs:28]: https://dev.azure.com/xamarin/public/_build/results?buildId=129239&view=results

Comment (anandhan-rajagopal) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue7396.cs:28]: Commited the images for windows UI tests.

**Review by StephaneDelcroix — APPROVED** [review]

**Review by jsuarezruiz — APPROVED** [review]

**Pull request merged**